ecommerce/baskets/views.py

```diff
@@ -1,3 +1,20 @@
-from django.shortcuts import render
+from rest_framework import viewsets
 
-# Create your views here.
+from core.mixins import DetailedViewSetMixin
+from baskets.filters import BasketFilter
+from baskets.models import Basket, BasketItem
+from baskets.serializers import BasketSerializer, BasketItemSerializer, BasketItemDetailedSerializer
+
+class BasketViewSet(viewsets.ModelViewSet):
+    queryset = Basket.objects.all()
+    serializer_class = BasketSerializer
+    filterset_class = BasketFilter
+
+class BasketItemViewSet(DetailedViewSetMixin, viewsets.ModelViewSet):
+    queryset = BasketItem.objects.all()
+    serializer_class = BasketItemSerializer
+
```

Comment:
- I don't know these detailed views works as expected but maybe you can add code blocks in your detailed views;
 ```py
serializer_action_classes = {
        "detailed_list": BasketDetailedSerializer,
        "detailed": BasketDetailedSerializer,
    }
```